tests/projects/test_project.py

```diff
@@ -47,6 +48,37 @@ def assets_path():
     return pathlib.Path(__file__).absolute().parent / "assets"
 
 
+@pytest.fixture
+def remote_builder_mock(monkeypatch):
+    class _MockDb:
```

Comment:
why setting the class inside the fixture? it can be done like our k8s secrets mock

Reply:
Fixed.